### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by Luke-Symes-Tsy**
A few fixes:

1. Update parameter template so that Accommodation Supplement parameters get merged in properly. Previously, only 1/4 of these were working and the rest were zero
2. Update App Parameters to HYEFU23
3. Update plot axis ticks to have nicer formatting
4. Update secondary earner UI to use TY24 minimum wage and "step" of 5 hours.
5. Change hover mode to show all components by default

### CODE REVIEW

**Review by MeghanStephensNZ — APPROVED** [review]

**Review by MeghanStephensNZ — APPROVED** [review]

**Review by MeghanStephensNZ — APPROVED** [review]

File: R/app_utilities.R [hunk 1]
```diff
@@ -134,84 +131,89 @@ amounts_net_plot <-
...
+      p <-
+      p %>% add_lines(data = Y, x = ~gross_wage1_annual,
                       y = ~Net_Income, name = 'Net Income', color = I("black"),
-                      hovertemplate = paste("Net Income: %{y:$,.0f}<extra></extra>")) 
+                      hovertemplate = paste("Net Income: %{y:$,.0f}<extra></extra>"))
+    
+    # Add secondary axis with hours
+    p <- p %>% add_trace(x=~hours1, y=~0, line=list(width = 0), xaxis="x2",
+              data=X, showlegend=FALSE, inherit=FALSE,
+              hoverinfo = "none", type = "scatter", mode = "lines") %>%
```
Comment (BenAtTsy) [inline R/app_utilities.R:216]: What does this pipe into?

File: ui.R [hunk 2]
```diff
@@ -37,8 +44,7 @@ shinyUI(fluidPage(
         column(6, numericInput(
           "wage1_hourly", "Hourly wage ($):",
           min = 18, max = 100,
-          #TY24 minimum wage
-          value = 22.70, step = .5,
+          value = DEFAULT_HOURLY_WAGE, step = .5,
```
Comment (BenAtTsy) [inline ui.R:47]: rogue comma?

**Review by BenAtTsy — CHANGES_REQUESTED** [review]

File: R/app_utilities.R (hunk 1 above)
Comment (Luke-Symes-Tsy) [reply R/app_utilities.R]: It pipes into the return(p) function; good catch

File: ui.R (hunk 2 above)
Comment (Luke-Symes-Tsy) [reply ui.R]: Yeah, I think it was already there, another good catch thanks

**Review by BenAtTsy — APPROVED** [review]

**Pull request merged**